client.go

```diff
@@ -12,12 +12,13 @@ import (
 	"time"
 
 	"github.com/go-resty/resty/v2"
+
 	"github.com/golang-jwt/jwt/v4"
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
 	"github.com/segmentio/ksuid"
 
-	"github.com/Nerzal/gocloak/v11/pkg/jwx"
+	"github.com/netapp-polaris/gocloak/v11/pkg/jwx"
```

Comment:
We shouldn't change that import here. 

If you need this change for a local redirect of the pkg import, you can make use of the "replace" directive in the go.mod file while developing.